model/versioning/Versioned.php

```diff
@@ -301,12 +308,13 @@ public function augmentSQL(SQLSelect $query, DataQuery $dataQuery = null) {
 		// Reading a specific stage (Stage or Live)
 		case 'stage':
 			$stage = $dataQuery->getQueryParam('Versioned.stage');
-			if($stage && ($stage != $this->defaultStage)) {
+			if($this->hasVersionedStage($stage) && $stage !== static::DRAFT_STAGE) {
```

Comment:
Do we have any examples of that @tractorcow? And wouldn't it need to be the `Live` stage, because the default read stage is Live?